monix-execution/shared/src/main/scala/monix/execution/FutureUtils.scala

```diff
@@ -161,6 +172,10 @@ object FutureUtils {
     p.future
   }
 
+  /** Creates a future that never completes. */
+  def never[A]: Future[A] =
+    Promise[A].future
```

Comment:
I only use this for testing. I could move this to the test suite, such that nobody will accidentally use it.